### PR TITLE
Add Account content when user is logged

### PR DESCRIPTION
**Pull request by brunojdo**
#### What is the purpose of this pull request?

Add Account content when the user is logged

https://github.com/vtex-apps/login/issues/5

#### What problem is this solving?

Add account options that will redirect the user to `store/account` to update your profile. 

#### Screenshots or example usage
https://login--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Login.js [hunk 1]
```diff
@@ -64,37 +73,38 @@ export default class Login extends Component {
     this.setState(state)
   }
 
+  handleAccountContent = () => {
+    this.setState({ step: 3 })
```
Comment (andreldsa) [inline react/Login.js]: When the page refresh the step reset.

File: react/components/AccountOptions.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+import React, { Component } from 'react'
+
+// Component that shows account options to the user.
+class AccountOptions extends Component {
+    render() {
+        return (
+            <h4>Bem vindo!</h4>
```
Comment (andreldsa) [inline react/components/AccountOptions.js]: i18n

File: react/Login.js [hunk 3]
```diff
@@ -64,37 +73,38 @@ export default class Login extends Component {
...
           onMouseEnter={() => this.handleUpdateState({ isMouseOnButton: true })}
-          onMouseLeave={() =>
-            this.handleUpdateState({ isMouseOnButton: false })
-          }
+          onMouseLeave={() => this.handleUpdateState({ isMouseOnButton: false })}
         >
+          {profile &&
+            (<div className="f7"
+              onMouseEnter={() => this.handleAccountContent()}>
+              Olá, {profile.firstName}
```
Comment (andreldsa) [inline react/Login.js]: i18n

File: react/components/CodeConfirmation.js [hunk 4]
```diff
@@ -21,22 +20,22 @@ class CodeConfirmation extends Component {
...
         variables: {
           authInput: { email, code }
         }
       }).then(
-        res => {
-          console.log(res)
-        },
-        err => {
+        ({ data }) => {
+          if (data.accessKeySignIn) {
```
Comment (andreldsa) [inline react/components/CodeConfirmation.js]: Add `data && ...` to avoid NPE

File: react/Login.js (hunk 1 above)
Comment (brunojdo) [reply react/Login.js]: Tomorrow, we discuss a design for this.

**Pull request merged**